node-backend/server.js

```diff
@@ -1,10 +1,13 @@
 const express = require("express");
```

Comment:
It's a good practice to `require` all `third-party` modules first and after a `line-break` all custom modules.

Reply:
sure